website/integrations: clarify Jellyfin LDAP bind permissions (#21975)
goauthentik/authentik: website/integrations/media/jellyfin/index.md

### authentik Configuration

Follow the [LDAP provider setup](/docs/add-secure-apps/providers/ldap/create-ldap-provider/) to create the LDAP application, provider, service account, and outpost. Complete the [service account creation](/docs/add-secure-apps/providers/ldap/create-ldap-provider/#create-a-service-account) and [LDAP search permission](/docs/add-secure-apps/providers/ldap/create-ldap-provider/#assign-the-ldap-search-permission-to-the-service-account) steps for the account Jellyfin uses to connect to LDAP.

> If the LDAP application has access restrictions, allow the LDAP service account through the application's [policy, group, or user bindings](/docs/add-secure-apps/applications/manage_apps/#use-bindings-to-control-access).

Reviewer: ```suggestion
If access to the authentik LDAP application is restricted, allow the LDAP service account access via the application's [policy, group, or user bindings](/docs/add-secure-apps/applications/manage_apps/#use-bindings-to-control-access).
```

### authentik Configuration

Follow the [LDAP provider setup](/docs/add-secure-apps/providers/ldap/create-ldap-provider/) to create the LDAP application, provider, service account, and outpost.

> The service account used as the Jellyfin LDAP bind user must be allowed to access the LDAP provider's application. If the application has policy, group, or user bindings, include the service account in those bindings.

Reviewer 2: "... include the service account in those bindings."

How exactly? It is the name of the service account, or.. what value? And where?

> Jellyfin searches the LDAP directory during setup and login, so the bind user also needs the **Search full LDAP directory** permission on the LDAP provider. Assign this permission from **Applications** > **Providers**, select the LDAP provider, open the **Permissions** tab, and assign **Search full LDAP directory** to the bind user or a role containing the bind user.

Reviewer 2: "user binding"? "bound user"? ... The phrase "bind user" doesn't sound right.

### Jellyfin configuration

> 1. Use the LDAP service account configured in authentik as the Jellyfin LDAP bind user.

Reviewer 2: Use new phrasing instead of "bind user".
